pyproject.toml

```diff
@@ -49,6 +49,7 @@ dependencies = [
 [project.urls]
 Homepage = "https://github.com/fastapi/fastapi"
 Documentation = "https://fastapi.tiangolo.com/"
+Changelog = "https://fastapi.tiangolo.com/release-notes/"
 Repository = "https://github.com/fastapi/fastapi"
```

Comment:
I also suggest adding an `Issues = "https://github.com/fastapi/fastapi/issues"` to not return to this a second time.

Reply:
Good idea ! I just added the suggested link